src/main/java/net/imglib2/algorithm/fill/FloodFill.java

```diff
@@ -90,6 +90,7 @@ public class FloodFill
 	 *            input pixel type
 	 * @param <U>
 	 *            fill label type
+	 * @implNote op name='image.floodFill', type=Computer2
```

Comment:
Let's move to `morphology`. That's where skimage puts flood fill.

Reply:
Good idea 👍